src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseSource.iOS.cs

```diff
@@ -743,6 +743,14 @@ public ContentControl Content
 
 					ContentView.AddSubview(value);
 
+
+					Layouter.ArrangeChild(value, new Rect(0, 0, (float)Frame.Width, (float)Frame.Height));
```

Comment:
@jeromelaban @dr1rrb  will doing this generate a lot of "noise"? This, combined with the `UpdateContentLayoutSlots()` works perfect for all the unit tests

Reply:
Humm, it's weird that we arrange child/content without even measuring it. I'm worried that we could arrange twice each item in the list. Does the `ClearMeasuredSize` below cause a layout pass?

Also looking at your comment, is it possible that we are bypassing that internal container during layouting ?